packages/core/pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "rpaframework-core"
-version = "10.0.5"
+version = "10.0.6"
```

```suggestion
version = "10.1.0"
```

I would increase the minor (not hotfix) version of it since this both extends a feature and changes the behaviour (in a non-breaking way) with:
```python
control = self._get_control_from_params(search_params)
        if auto.ControlsAreSame(control, root_control):
            return Control.CreateControlFromControl(root_control)
```